### PR TITLE
IMPROVEMENT_Add_datatable

### PR DESCRIPTION
**Pull request by ioigoume**
Added Javascript Datatable plugin and associated js/css libraries.

### CODE REVIEW

File: www/services.php [hunk 1]
```diff
@@ -112,7 +112,6 @@
 function compareByName($a, $b) {
   return strcmp($a["name"], $b["name"]);
 }
-usort($serviceList, 'compareByName');
```
Comment (NicolasLiampotis) [inline www/services.php:115]: I would prefer to leave this line here for templates that don't support sorting

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Minor changes and a couple of questions:
- I got a 404 file not found error for `baseURL/rciaminfo/res/images/sort_asc.png`. Should this image be added to the resources?
- Is there a way to configure the number of entries shown per page; e.g. add this as an option to `config/module_rciaminfo.php`?

**Comment by ioigoume** [comment]
Made the changes and updated the PR.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**